workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/layout-page-templates/display-page-templates/learn-article/display-page-template.json

```diff
@@ -1,10 +1,5 @@
 {
-	"contentSubtype": {
-		"subtypeKey": "LEARN-ARTICLE"
-	},
-	"contentType": {
-		"className": "com.liferay.journal.model.JournalArticle"
-	},
-	"defaultTemplate": true,
+	"contentSubtype": {"subtypeId": 32194},
```

Comment:
@rtienpal I dont think we want to link the ID, since it will be generated differently every time. Ideally we can still use the subtypeKey, if we can get it to work

Reply:
Just sent a new commit, adjusting this